src/BuildScriptGeneratorCli/SettingsKeys.cs

```diff
@@ -68,5 +68,6 @@ public static class SettingsKeys
         public const string DisableRecursiveLookUp = "DISABLE_RECURSIVE_LOOKUP";
         public const string CustomRequirementsTxtPath = "CUSTOM_REQUIREMENTSTXT_PATH";
         public const string DebianFlavor = "DEBIAN_FLAVOR";
+        public const string CallerId = "USER_CALLER_ID";
```

Comment:
We need to also change the environment variable itself to be `CALLER_ID`

Reply:
In the case, our partners need to aware of that as well. isn't it? They have to send the env variable `CALLER_ID`. how this works?

Reply:
Correct they have to set the `CALLER_ID` environment variable to some value, but in this file, the environment variable we'll be reading is `USER_CALLER_ID` since we didn't change it yet after the comment from Dan to drop the `USER` part of it.